linux_os/guide/system/permissions/mounting/kernel_module_fat_disabled/rule.yml

```diff
@@ -0,0 +1,28 @@
+documentation_complete: true
+
+prodtype: rhel6,rhel7,rhel8,fedora,rhv4
+
+title: 'Disable Mounting of FAT filesystems'
+
+description: |-
+    {{{ describe_module_disable(module="vfat") }}}
+    This effectively prevents usage of this uncommon filesystem.
+
+rationale: |-
+    Linux kernel modules which implement filesystems that are not needed by the
+    local system should be disabled.
+
+severity: low
+
```

Comment:
Could CCEs also be added? A general template can be found in the other kernel_* rules.

Essentially, grab CCEs from https://github.com/ComplianceAsCode/content/blob/master/shared/references/cce-redhat-avail.txt and assign them :) 

There should be assignments for each of the RHEL versions. Haven't started with RHV yet.

Reply:
See https://github.com/ComplianceAsCode/content/pull/4819 for example of assigning CCEs. That PR starts from the bottom of the available CCEs, so if you begin assigning from the top, there should not be any PR conflicts.